proposals/image-planes/README.md

```diff
@@ -0,0 +1,175 @@
+```
+##### Which would generate this .usda:
+```
+def Xform "world" {
+    def Camera "cam" (        
+        apiSchemas = ["UsdGeomImagePlaneAPI:imagePlane1"]
+    ){
+        ...
+        string[] imagePlanes = ["imagePlane1"]
+        asset imagePlane1:image = {
+            1001: @/path/to/image.1001.exr@, 
+            1002:...
+            }
+        float imagePlane1:depth = 1000.0
+    }
+}
+```
+
+## Hydra Implementation
+TODO: Flesh out and discuss with Hydra team.
+
+Ideas:
+- Use UsdPreviewSurface textures in a way similar to GeomModel texture cards.
+- Do the compositing in an Hdx post task
+
+## Use Cases
+### Hydra / Usdview
+It would be helpful to be able to view CG elements against a backdrop of the production plate when checking exported usd caches.
+
+### DCC Interchange
+Having a "UsdLux" style standard for Image Planes would be very useful when implementing importers and exporters for DCCs like Maya, Blender, Nuke, etc.
+
+## Alternative Implementation Strategies
+### 1.) As a concrete prim type
+Here is a working implementation that was done before Multi-Apply schemas were added to USD.
+
+It's a fully featured solution that is based around Autodesk Maya's "underworld" image plane nodes where many planes 
+can live under a camera. This specific implementation is just a quick attempt to move maya image planes across packages 
+and the hydra implementation is just a workaround that generates an HdMesh rprim (instead of creating a new image plane 
+prim type).
+
+Reference internal PR from Luma: https://github.com/LumaPictures/USD/pull/1
+
+#### Pros:
+- Prim attributes like [visibility, purpose] can be leveraged to provide intuitive functionality
+- Easy to see Image Planes in a hierarchy
+#### Shortfalls:
+- Requires relationships between camera + plane prims
+- Requires an extra API schema to encode the relationship on the Camera.
+- There's the possibility that someone targets something other than an ImagePlane with the relationship, so there are more ways to "get stuff wrong". 
+
+### 2.) Using existing USD types (Plane + material + camera + expressions)
+- It is possible to create a UsdPlane with a material and accomplish the same thing as an image plane.
+#### Pros:
+- No need for this proposal :)
+#### Shortfalls:
+- Requires users to implement complex authoring code.
+- No explicit identity as an "Image Plane". This will make it hard to successfully roundtrip from DCCs to USD.
+
+## Appendix: "Fit" Strategy Discussion
+One of the key decisions for this schema is to come up with a universally acceptable way to describe how the
+image plane will be positioned in relation to the camera film back. We want something that will cover all use cases
+without being to burdensome to describe.
+
+In our discussions, we determined fit should be relative to the film back as opposed to in scene space or render space.
+The exact form this takes is still up for discussion...
```

Comment:
I would love to hear others weigh in on this!  The approach taken here is similar to that taken in formulating the UsdTransform2D shader as a low-level texCoord transform, rather than something more artist-friendly relating directly to the controls they'd see in their DCC.

One could argue that this schema is more directly "user-facing" than the details of placing textures in a UsdShade network, so maybe a well-defined set of artist-level choices *is* more appropriate, here?  But the Maya schema forces you out to a different set of controls if you need something more boutique, including rotations...